test/hydration-test.js

```diff
@@ -145,7 +149,7 @@ describe('Hydration', function() {
     expect(hydration.calculateFirstWeekOunces(userRepo, 4)[6]).to.eql('2019/04/15: 36');
   });
 
-  it('should find sleep quality by day for that days week', function() {
+  it('should calculate random week ounces for a user', function() {
```

Comment:
Thank you for rephrasing the test, now it matches the actual test which  checks hydration.